types/three/examples/jsm/webxr/XRButton.d.ts

```diff
@@ -0,0 +1,5 @@
+import { WebGLRenderer } from '../../../src/Three';
+
+export namespace XRButton {
+    function createButton(renderer: WebGLRenderer): HTMLElement;
+}
```

Comment:
Can you make this a class with a static method?

Reply:
Sure!

Reply:
@Methuselah96 I've changed it into a class and update the tests. When I run yarn test, I get the below error
'no-unnecessary-class  Every member of this class is static. Use namespaces or plain objects instead.'
Let me know how I should resolve.
Thanks!

Reply:
Huh, interesting. That seems like a strange lint rule to have for declaration files and seems to have more to do with original code. I would disable with a comment right above the class declaration with `// tslint:disable-next-line:no-unnecessary-class`.